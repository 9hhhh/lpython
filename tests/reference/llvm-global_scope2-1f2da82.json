{
    "basename": "llvm-global_scope2-1f2da82",
    "cmd": "lfortran --show-llvm {infile} -o {outfile}",
    "infile": "tests/global_scope2.f90",
    "infile_hash": "e511f118b9a040b5e7a3ff2ad75cd7942dee08d3a21bd4ca5cd668b3",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "llvm-global_scope2-1f2da82.stdout",
    "stdout_hash": "b09b017d24ffbfc71428d41cef0b42cdd8492771ad952cb229167613",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}