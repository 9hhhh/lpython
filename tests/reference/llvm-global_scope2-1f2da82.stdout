; ModuleID = 'LFortran'
source_filename = "LFortran"

@x = external global i64

define i64 @f() {
.entry:
  %f = alloca i64
  store i64 6, i64* @x
  %0 = load i64, i64* %f
  ret i64 %0
}

